docs/backend/behaviors.md

```diff
@@ -11,12 +11,305 @@ myst:
+Once a behavior has been installed, it can be applied to any content type by selecting it in the Dexterity configuration section of the Plone control panel.
+This allows the object to gain the additional functionality provided by the behavior.
+
+Overall, behaviors are an important part of the Plone content management system and allow for powerful customization and extensibility of content objects.
+
+## Built-in behaviors
+
+| short name  | Title            | Desription                    |
+|-------------|------------------|-------------------------------|
+| plone.allowdiscussion  | Allow discussion | Allow discussion on this item |
+| plone.basic  | Basic metadata | Adds title and description fields. |
+| plone.categorization  | Categorization | Adds keywords and language fields. |
+| plone.collection  | Collection | Adds collection behavior |
+| plone.publication  | Date range | Adds effective date and expiration date fields. |
+| plone.dublincore | Dublin Core metadata | Adds standard metadata fields (equals Basic metadata +    Categorization + Effective range + Ownership  |
+| plone.eventattendees  | Event Attendees | Attendees extension for Events. |
+| plone.eventbasic  | Event Basic | Basic Event schema. |
+| plone.eventcontact  | Event Contact | Contact extension for Events. |
+| plone.eventlocation  | Event Location | Location extension for Events. |
+| plone.eventrecurrence  | Event Recurrence | Recurrence extension for Events. |
+| plone.excludefromnavigation  | Exclude From navigation | Allow items to be excluded from navigation |
+| plone.constraintypes| Folder Addable Constrains | Restrict the content types that can be added to folderish   content |
+| plone. textindexer | Full-Text Indexing | Enables the enhanced full-text indexing for a content type. If a field in the schema is marked with the `searchable` directive, its content gets added to the    `SearchableText` index in the catalog |
+| plone.leadimage  | Lead Image | Adds image and image caption fields |
+| plone.locking  | Locking | Locking support for dexterity |
+| plone.translatable  | Multilingual Support | Make this content type multilingual aware. Multilingual support must be installed. |
+| plone.namefromfilename | Name from file name | Automatically generate short URL name for content based on its    primary field file name
+| plone.namefromtitle | Name from title | Automatically generate short URL name for content based on its initial    title
+| plone.navigationroot  | Navigation root | Make all items of this type a navigation root |
+| plone.nextpreviousenabled  | Next previous navigation | Enable next previous navigation for all items of this type |
+| plone.nextprevioustoggle  | Next previous navigation toggle | Allow items to have next previous navigation enabled |
+| plone.ownership  | Ownership | Adds creator, contributor, and rights fields. |
```

Comment:
```suggestion
| plone.ownership  | Ownership | Adds creator, contributor, and rights fields. |
| plone.____  | Preview Image | Preview image for listings |
| plone.____  | Preview Image Link | Preview image for listings based on links |
```

Reply:
This are `volto.previewimage` and `volto.preview_image_link`.